src/lib.rs

```diff
@@ -149,6 +149,7 @@ pub trait AsyncConnection: SimpleAsyncConnection + Sized + Send {
     /// The argument to this method and the method's behavior varies by backend.
     /// See the documentation for that backend's connection class
     /// for details about what it accepts and how it behaves.
+    #[cfg(not(target_arch = "wasm32"))]
     fn establish(database_url: &str) -> impl Future<Output = ConnectionResult<Self>> + Send;
```

Comment:
I would like to keep this method in all configuration. 
It should be possible to provide an implementation for this for `wasm32` as you somehow need to establish a connection there as well, right?